sips/sip-02x/sip-02x-stacks-pay.md

```diff
@@ -0,0 +1,343 @@
+
+- operation: 'support'
+
+- recipient: A valid Stacks address.
+
+- If the token parameter is not 'STX', the functionName defaults to 'transfer' by the wallet.
+
+**Optional Parameters:**
+
+- token: Either 'STX' or a valid SIP-010 contract address. If not provided, defaults to 'STX'.
+
+- description: MAY be included to provide additional context for the payment request.
+
+- expiresAt: MAY be included; wallets MUST NOT process payment requests past the expiry time.
+
+- memo: MAY be included; MUST NOT include personal information.
+
+**Parameters to Ignore:**
+
+- amount, contractName, dueDate, functionName, invoiceNumber. Wallets MUST ignore these parameters if present.
+
+### `invoice`
+
+**Function:** Represents a payment request, such as an invoice for P2P payments or product purchases.
+
+**Required Parameters:**
+
+- operation: 'invoice'
+
+- recipient: A valid Stacks address.
+
+- token: MUST be either 'STX' or a valid SIP-010 contract address.
+
+- amount: The amount to be paid.
+
+**Optional Parameters:**
+
+- description: MAY be included to provide context for the payment request.
+
+- expiresAt: MAY be included; if included, wallets MUST NOT process payment requests past the expiry time.
+
+- invoiceNumber: MAY be included as an identifier.
+
+- dueDate: MAY be included.
+
+- memo: MAY be included. MUST NOT include personal information.
+
+**Parameters to Ignore:**
+
+contractName, functionName: Wallets MUST ignore these parameters if present.
+
+### `mint`
+
+**Function:** Represents a non-fungible token minting request.
+
+**Required Parameters:**
+
+- operation: 'mint'
+
+- contractName: A valid SIP-009 contract address.
+
+- functionName: A public contract function, typically `mint` or `claim`
+
+- token: MUST be either 'STX' or a valid SIP-010 contract name.
+
+- amount: The amount to be paid.
+
+**Optional Parameters:**
+
+- recipient: MAY be included; if present, MUST be a valid Stacks address. The NFT will be minted for the specified address.
+
+- description: MAY be included to provide context for the payment request
+
+- expiresAt: MAY be included. If included wallets MUST NOT process links after the expiry date/time.
+
+- invoiceNumber: MAY be included by the link creator to track individual requests.
+
+- memo: MAY be included. MUST NOT include personal information.
+
+**Parameters to Ignore:**
+
+- contractName, functionName, dueDate: Wallets MUST ignore these parameters if present.
+
+### Custom Operations
+
+Applications **MAY** define custom operations for specific use cases. Custom operation types **MUST** be prefixed to prevent naming conflicts (e.g., `'custom-example'`).
+
+- **Handling Unrecognized Operations**: If a wallet encounters an unrecognized operation type, it **SHOULD**:
+
+- **Warn the User**: Inform the user that the operation type is unrecognized.
+
+- **Provide Safe Defaults**: Default to a standard payment flow if possible.
+
+- **Fail Gracefully**: Prevent unexpected behavior or security risks.
```

Comment:
Felt like this made it more readable

```suggestion
### Custom Operations

Applications **MAY** define custom operations for specific use cases.

Custom operation types **MUST** be prefixed to prevent naming conflicts (e.g., `'custom-example'`).

**Handling Unrecognized Operations**: If a wallet encounters an unrecognized operation type, it **SHOULD**:

- **Warn the User**: Inform the user that the operation type is unrecognized.

- **Provide Safe Defaults**: Default to a standard payment flow if possible.

- **Fail Gracefully**: Prevent unexpected behavior or security risks.
```